packages/@tinacms/scripts/src/index.ts

```diff
@@ -179,9 +180,8 @@ e.g: "forestry types:gen --help"
 
 const config = (cwd = '') => {
   return {
-    mode: 'jit',
     // prefix: 'tina-',
-    important: true,
+    important: '.tina-tailwind',
```

Comment:
Looking through the styles, this causes every rule to preface with `.tina-tailwind`, not unlike the nesting we were doing.